### ISSUE TITLE
Using BPOG vs FPOG for Peak Saccade Velocity

### ISSUE THREAD

**Issue by ashkjones** [TheD2Lab/gazepoint-data-analysis#38]
## Definitions (From GazePoint API Documentation)
- _**POG**_: Point of gaze
- _**BPOG**_: The 'best' of the left/right eye POG data, which is the average of the left eye and right eye POG if both are available, or if now, then of either the left or right eye, depending on which one is valid. For most applications, use FPOG which includes fixation filtering of the BPOG data.
  - _[Ashley Note]: We filter out the lines of data where either eye is not available in the analysis code. Assume BPOG is always the average of left and right POG._
- _**BPOGX, BPOGY**_:  the X- and Y-coordinates of the best eye POG.
- _**FPOG**_: the Fixation POG data provides the user's point-of-gaze as determined by the internal fixation filter.
- _**FPOGX, FPOGY**_: the X- and Y-coordinates of the fixation POG.
- _**FPOGV**_: The valid flag with value of 1 (TRUE) if the fixation POG data is valid, and 0 (FALSE) if it is not. FPOGV valid is TRUE ONLY when either one, or both, of the eyes are detected AND a fixation is detected. FPOGV is FALSE all other times, for example when the subject blinks, when there is no face in the field of view, when the eyes move to the next fixation (i.e. a saccade). 
  - _[Ashley Note]: Due to the filtering process, we can assume that FPOGV = 0 indicated the data point occurs during a saccade._

## Problem
The saccade velocity calculation in our code rely on FPOG values. However, BPOG values may be more suitable.

https://github.com/TheD2Lab/gazepoint-data-analysis/blob/044cebff731f5ec4a94d17074486cb0757dcb92f/src/analysis/SaccadeVelocity.java#L7-L12

## Discussion
The FPOGX and FPOGY are rolling averages of the BPOGX and BPOGY values. From GazePoint Support:

> ...FPOG data is calculated from averaging the BPOG data, with a reset once the centroid is displaced indicating the fixation has ended and the eyes have moved to a new fixation. This is modeled on the position variance technique shown in the reference below.
> [REF] R. Jacob, Virtual Environments and Advanced Interface Design. New York, NY, USA: Oxford University Press, 1995, ch. Eye tracking in advanced interface design, pp. 258–288.

Jacob's (1995) focus is removing the jitter from fixation data to improve gaze controlled user interfaces. From Jacob (1995):
> We then identify and quickly report the start and approximate position of each recognized fixation. We ignore any reports of eye position during saccades themselves, since they are difficult for the eye tracker to catch and their dynamics are not particularly meaningful to the user-computer dialogue.

It seems that the fixation filtering algorithm is optimized for determining the x- and y-coordinates for fixation, but not for saccades.  GazePoint provides FPOGX and Y for data points associated with saccades, but it is not clear if FPOG values accurately describe the location of the participant's actual gaze during a saccade.

**This opens up the discussion of whether our analysis code should use BPOG data instead.** BPOG can be thought of the raw location of where the eye tracker detects the user's gaze.

## TLDR
Should BPOG be used over FPOG when calculating peak saccade velocity? This is an easy fix, but I want approval before I make the change.

**Comment by ashkjones** [comment]
Note to future readers, it was decided we will stick with FPOG.
